scripts/shared/lib/shflags

```diff
@@ -9,8 +9,8 @@ ensure_shflags() {
     fi
 
     if [ ! -f .shflags ]; then
-        echo "Downloading shflags ${SHFLAGS_VERSION:=1.2.3}"
-        if ! (curl -L "https://github.com/kward/shflags/archive/v${SHFLAGS_VERSION}.tar.gz" | tar -xzf - shflags-${SHFLAGS_VERSION}/shflags &&
+        echo "Downloading shflags ${SHFLAGS_VERSION}"
+        if ! (curl -L "https://github.com/kward/shflags/archive/${SHFLAGS_VERSION}.tar.gz" | tar -xzf - shflags-${SHFLAGS_VERSION}/shflags &&
               mv shflags-${SHFLAGS_VERSION}/shflags .shflags); then
```

Comment:
```suggestion
        if ! curl -L "https://raw.githubusercontent.com/kward/shflags/${SHFLAGS_VERSION}/src/shflags" > .shflags; then
```

Reply:
Only problem is for later versions this path is different.. :/